### DISCUSSION TITLE
Ping quelqu’un lors d’une vente

### DISCUSSION THREAD

**Discussion by OrionYp** [Crownicles/Crownicles#2568] [Nouvelles idées]
Ajouter une option à la commande de vente d'animal pour pouvoir choisir l'acheteur et éviter qu'un inconnu arrive et fasse l'achat à sa place.

**Comment by BastLast** [comment]
> deleted

Merci de faire ce genre de retour sur le discord officiel du bot @Carsebel

**Comment by Cirpacha** [comment]
Très bonne idée, ça éviterait les possibles abus de joueurs malveillants
En plus c'est déjà le cas pour l'échange de familiers
C'est cool de te voir garder autant d'intérêt pour le bot, il n'est jamais trop tard pour prendre un nouveau départ 👍
